examples/custom_hint/src/main.rs

```diff
@@ -1,30 +1,36 @@
-use cairo_vm::cairo_run::cairo_run;
+#![allow(unused_imports)]
+
+use cairo_vm::cairo_run::{cairo_run, CairoRunConfig};
+use cairo_vm::felt::Felt252;
 use cairo_vm::hint_processor::builtin_hint_processor::builtin_hint_processor_definition::{
     BuiltinHintProcessor, HintFunc,
 };
 use cairo_vm::hint_processor::builtin_hint_processor::hint_utils::get_integer_from_var_name;
 use cairo_vm::hint_processor::hint_processor_definition::HintReference;
-use cairo_vm::types::exec_scope::ExecutionScopes;
 use cairo_vm::serde::deserialize_program::ApTracking;
+use cairo_vm::types::exec_scope::ExecutionScopes;
 use cairo_vm::vm::{errors::hint_errors::HintError, vm_core::VirtualMachine};
-use num_bigint::BigInt;
 use std::collections::HashMap;
+use std::fs::File;
+use std::io::{BufReader, Read};
 use std::path::Path;
 use std::rc::Rc;
 
 // Create the function that implements the custom hint
+#[cfg(feature = "std")]
 fn print_a_hint(
     vm: &mut VirtualMachine,
     _exec_scopes: &mut ExecutionScopes,
     ids_data: &HashMap<String, HintReference>,
     ap_tracking: &ApTracking,
-    _constants: &HashMap<String, BigInt>,
+    _constants: &HashMap<String, Felt252>,
 ) -> Result<(), HintError> {
     let a = get_integer_from_var_name("a", vm, ids_data, ap_tracking)?;
     println!("{}", a);
     Ok(())
 }
 
+#[cfg(feature = "std")]
```

Comment:
This `#[cfg]` may be a bit confusing. I think we can avoid it by adding a [`required-features`](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-required-features-field) field to the `Cargo.toml` target:
```
[[example]]
...
required-features = ["std"]
```

Reply:
Applied change. Although we have that tag in many places because of the `no_std` (so it shouldn't be confusing), I agree that is way cleaner to have it in the `Cargo.toml`, since the example is not supposed to run without `std` ever